### PR TITLE
BaSP-TG: employee get and post

### PR DESCRIPTION
**Pull request by Romimonde**
:)

### CODE REVIEW

**Review by javierperezraffo — DISMISSED** [review]

File: src/resources/employees.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  const foundEmp = employees.find((employee) => employee.id === empId);
+
+  if (foundEmp) {
+    res.send(foundEmp);
+  } else {
+    res.send('Employee not found');
+  }
+});
+
+router.get('/getFilteredList/:name/:lastName/:phone/:email', (req, res) => {
```
Comment (Ignacio-Sormani) [inline src/resources/employees.js:18]: Use query params to make filters:

'/getFilteredList?name=ignacio&lastName=sormani&....'

File: src/resources/employees.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  let arrEmps = employees;
+  const empName = req.params.name;
```
Comment (Ignacio-Sormani) [inline src/resources/employees.js:20]: You can do:

const {
   empName,
   empLastName,
   ...
} = req.query;

So you get all query params into const at once

**Review by renzopimentel — DISMISSED** [review]

File: src/data/employees.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
+[
+    {
+        "id":"1",
```
Comment (Ignacio-Sormani) [inline src/data/employees.json:3]: Update ids so they are a string of 7 digits

File: src/resources/employees.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  employees.push(newEmp);
+  fs.writeFile('src/data/employees.json', JSON.stringify(employees), (err) => {
+    if (err) {
+      res.status(400).send('Cannot add new employee');
+    } else {
+      res.status(201).send('Employee Created');
+    }
+  });
+});
+module.exports = router;
```
Comment (Ignacio-Sormani) [inline src/resources/employees.js:58]: Add one blank line between lines 57 and 58

**Review by renzopimentel — APPROVED** [review]

**Pull request merged**